src/snowflake/snowpark/query_history.py

```diff
@@ -37,11 +37,13 @@ def __init__(
         session: "snowflake.snowpark.session.Session",
         include_describe: bool = False,
         include_thread_id: bool = False,
+        record_error_queries: bool = False,
```

Comment:
as a second thought, include_error fits better with other names. Maybe we can also add small doc string saying

```
    def include_describe(self) -> bool:
    """Record QueryRecords for describe queries into this listener."""
```